packages/gateway/src/config/routes.ts

```diff
@@ -43,7 +45,7 @@ export const getRoutes = () => {
       method: 'GET',
       path: '/ping',
       handler: healthCheckHandler,
-      config: {
+      options: {
```

Comment:
Were config and options just aliases?

Reply:
Maybe they were called config previously and it still worked that way but according to the types it should be `options`